user-impl/src/main/java/com/example/auction/user/impl/UserEntity.java

```diff
@@ -44,7 +44,7 @@ private Behavior notCreated() {
         );
 
         b.setCommandHandler(CreateUser.class, (create, ctx) -> {
-            User user = new User(UUID.fromString(entityId()), create.getName());
+            User user = new User(UUID.fromString(entityId()), create.getName(),"admin","admin");
```

Comment:
@jroper Do you to store password in a separate table or in the same table you want to store the encrypted password?